pandas/io/packers.py

```diff
@@ -38,39 +38,37 @@
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 """
 
-from datetime import datetime, date, timedelta
-from dateutil.parser import parse
 import os
-from textwrap import dedent
 import warnings
+from datetime import datetime, date, timedelta
+from textwrap import dedent
 
 import numpy as np
+from dateutil.parser import parse
+
+from pandas import (Timestamp, Period, Series,  # noqa
+                    Index, MultiIndex, RangeIndex, PeriodIndex, DatetimeIndex,
+                    NaT,
+                    Categorical, IntervalIndex, Interval)
+
+from pandas.util._move import (
+    BadMove as _BadMove,
+    move_into_mutable_buffer as _move_into_mutable_buffer,
+)
 from pandas import compat
 from pandas.compat import u, u_safe
-
+from pandas.core import internals
+from pandas.core.arrays import IntervalArray
 from pandas.core.dtypes.common import (
     is_categorical_dtype, is_object_dtype,
     needs_i8_conversion, pandas_dtype)
-
-from pandas import (Timestamp, Period, Series, DataFrame,  # noqa
-                    Index, MultiIndex, Float64Index, Int64Index,
-                    Panel, RangeIndex, PeriodIndex, DatetimeIndex, NaT,
-                    Categorical, CategoricalIndex, IntervalIndex, Interval,
-                    TimedeltaIndex)
-from pandas.core.arrays import IntervalArray
+from pandas.core.generic import NDFrame
+from pandas.core.internals import BlockManager, make_block, _safe_reshape
 from pandas.core.sparse.api import SparseSeries, SparseDataFrame
 from pandas.core.sparse.array import BlockIndex, IntIndex
-from pandas.core.generic import NDFrame
 from pandas.errors import PerformanceWarning
```

Comment:
pandas.errors is among the zero-dependency directories id put somewhere around util/compat

Reply:
Updated thank!